api/metadata_handler.go

```diff
@@ -51,21 +51,22 @@ func apiMetadataTriageHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	wptfyiBotClient, err := shared.NewGitHubClientFromToken(ctx, "github-wpt-fyi-bot-token")
+	botToken, err := shared.GetSecret(ds, "github-wpt-fyi-bot-token")
 	if err != nil {
 		http.Error(w, "Unable to get GitHub client for wpt-fyi-bot: "+err.Error(), http.StatusInternalServerError)
```

Comment:
Two things here
- maybe "Unable to get the GitHub OAuth token for wpt-fyi-bot:"?
- I know this should be asked early - is it safe to return a http.Error? It might be leaking information related to the secret?

Reply:
Both of these are addressed by #2167